src/main/java/pl/envelo/moovelo/entity/events/InternalEvent.java

```diff
@@ -0,0 +1,51 @@
+package pl.envelo.moovelo.entity.events;
+
+import lombok.Getter;
+import lombok.Setter;
+import pl.envelo.moovelo.entity.Comment;
+import pl.envelo.moovelo.entity.Hashtag;
+import pl.envelo.moovelo.entity.groups.Group;
+import pl.envelo.moovelo.entity.actors.BasicUser;
+import pl.envelo.moovelo.entity.surveys.EventSurvey;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Set;
+
+@Entity
+@Inheritance(strategy = InheritanceType.JOINED)
+@Getter
+@Setter
+public class InternalEvent extends Event {
+
+    @ManyToOne
+    private Group group;
+
+    @OneToMany
+    private List<BasicUser> invited;
```

Comment:
W klasie InternalEvent nie ma pola invited, jest za to w klasie Event ->  List<<BasicUser>> usersWithAccess